src/Veldrid/D3D11/D3D11GraphicsDevice.cs

```diff
@@ -113,6 +130,31 @@ public D3D11GraphicsDevice(GraphicsDeviceOptions options, SwapchainDescription?
             PostDeviceCreated();
         }
 
+        private static bool SdkLayersAvailable()
```

Comment:
Can you cache the result of this function so that the initial evaluation only needs to happen once? It involves creating a whole dummy Device, so it's fairly expensive.

Reply:
@mellinoe  fixed